src/App.tsx

```diff
@@ -1,25 +1,48 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [selectedPerPageSelector, setSelectedPerPageSelector] = useState('5');
```

Comment:
The `selectedPerPageSelector` state is initialized as a string ('5'). Since it is used as a number in calculations and passed to the `Pagination` component, consider initializing it as a number (e.g., `useState(5)`) to maintain type consistency.